### ISSUE TITLE
Upgrade Parser dependency to 2.6

### ISSUE THREAD

**Issue by snasirca** [deep-cover/deep-cover#55]
My project depends on the `parser` gem being greater than version `2.6`. `deep-cover` depends on `2.5`. So I'm having trouble installing `deep-cover`. I tried to fork the repo and upgrade the dependency myself, but then I have to also upgrade the `rubocop` version that `deep-cover` uses. This is because `deep-cover` is using an outdated version of `rubocop` that no longer supports `parser@2.5`.

I can get the `rspec` command to pass upgrading to `parser@2.6` but `rubocop` gives me a ton of offenses (27). I've made these changes to the `.rubocop.yml` file as per the warnings about outdated configurations:

```yaml
Layout/IndentFirstArgument:
  IndentationWidth: 4

Layout/IndentFirstArrayElement:
  EnforcedStyle: align_brackets

Layout/IndentFirstHashElement:
  EnforcedStyle: align_braces
```

I imagine I'd need to make more changes but what I'd like to see is if anyone else is attempting this and/or has strong opinions about how Rubocop for this repo should be configured.

Attaching all the offenses: [rubocop_failures.txt](https://github.com/deep-cover/deep-cover/files/3444495/rubocop_failures.txt)

**Comment by marcandre** [comment]
Sorry for these issues. It's a pity that rubocop is a dependency that is so fickle we need to get it locked. Let me have a look at this...

**Comment by marcandre** [comment]
So I updated to the latest RuboCop and we are now compatible with `parser` 2.6.

Released as [v0.7.5](https://github.com/deep-cover/deep-cover/releases/tag/v0.7.5)

Thanks for opening the issue.
